Automattic/woocommerce-payments
# chore: remove giropay-specific settings page

Fixes #

#### Changes proposed in this Pull Request









With UPE we don't need payment method-specific settings pages anymore.
Removing giropay, keeping `1-click checkouts` (for now)

#### Testing instructions





- Ensure you have the "grouped settings" flag enabled in WCPay Dev Tools
- Visit http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout
- Click "customize appearance" under "1-click checkouts"
- Page appears
- http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout&section=woocommerce_payments_giropay shows an error message

-------------------
